src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import static core.basesyntax.Shape.ISOSCELES_TRAPESOID;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int lowBase;
+    private int topBase;
+    private int hight;
+
+    public IsoscelesTrapezoid(int lowBase, int topBase, int hight, Color color) {
+        super(color);
+        this.lowBase = lowBase;
+        this.topBase = topBase;
+        this.hight = hight;
+    }
+
+    public int getLowBase() {
+        return lowBase;
+    }
+
+    public int getTopBase() {
+        return topBase;
+    }
+
+    public int getHight() {
+        return hight;
+    }
+
+    @Override
+    public Shape getName() {
+        return ISOSCELES_TRAPESOID;
```

Comment:
```suggestion
    @Override
    public Shape getName() {
        return getClass().getSimpleName();
```
let's do this instead in all figures